restaurant/my_restaurant/models.py

```diff
@@ -1,3 +1,81 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator, MaxValueValidator
 
 # Create your models here.
+
+
+class Menu(models.Model):
+    pass
+
+
+class Location(models.Model):
+    name = models.CharField(max_length=50)
+
+
+class Restaurant(models.Model):
+    VEGAN = "Vegan"
+    VEGETARIAN = "Veg"
+    NON_VEGETARIAN = "Non-veg"
+
+    TYPE_CHOICES = [
+        (VEGAN, "Vegan"),
+        (VEGETARIAN, "Vegetarian"),
+        (NON_VEGETARIAN, "Non-Vegetarian"),
+    ]
+
+    name = models.CharField(max_length=100)
+    type = models.CharField(max_length=7, choices=TYPE_CHOICES)
+    opening_time = models.TimeField()
+    closing_time = models.TimeField()
+    cost_for_two = models.CharField(max_length=4)
+    menu = models.ForeignKey(Menu, on_delete=models.CASCADE)
+    door_no = models.CharField(max_length=7)
+    street = models.CharField(max_length=50)
+    location = models.ForeignKey(Location, on_delete=models.CASCADE)
+    pincode = models.CharField(max_length=6)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6)
+
+
+class Item(models.Model):
+    name = models.CharField(max_length=50)
+    whether_veg = models.BooleanField()
```

Comment:
```suggestion
    type = models.BooleanField()
```

Should be choices of veg, non veg, vegan